woocommerce/grow
# Get Plugin releases Action: Fix error counting RC releases

### Changes proposed in this Pull Request:

Closes #65 

RC was not being counted as the output for plugins other than WordPress. So when we added Including RC option they were adding more releases than expected. 


### Detailed test instructions:


1.  Check if the [Pre Release is Compiling](https://github.com/woocommerce/grow/actions/runs/5621686842) 
2.  Check if the [action outputs make sense](https://github.com/woocommerce/grow/actions/runs/5621821892). 


### Additional details:


### Changelog entry

> Fix - Error counting RC releases.
